--- liferea-1.10.2/plugins/media-player.py~	2013-08-01 05:29:12.000000000 +0800
+++ liferea-1.10.2/plugins/media-player.py	2013-09-08 06:50:18.878885039 +0800
@@ -10,9 +10,6 @@
 from gi.repository import Gst
 
 # FIXME: Upgrade to 0.11
-#import gi
-#gi.require_version('Gst', '0.11')
-#from gi.repository import Gst
 
 class MediaPlayerPlugin(GObject.Object, Liferea.MediaPlayerActivatable):
     __gtype_name__ = 'MediaPlayerPlugin'
